src/modules/transaction/components/TransactionDetailRow/TransactionDetailRow.css

```diff
@@ -0,0 +1,167 @@
+@import '../../../../../setup/react/app/mixins.css';
+
+.container {
+  display: flex;
+  width: 100%;
+  position: relative;
+
+  & > div:first-child {
+    padding: 20px 22px 20px 0px;
+    color: var(--color-content-light);
+    font-weight: var(--font-weight-normal);
+    font-size: 14px;
+  }
+
+  & > .eventAction {
+    padding: 0px;
+  }
+
+  &::before {
+    content: '';
+    min-width: 100%;
+    position: absolute;
+    height: 1px;
+    border-top: 1px var(--color-platinum) solid;
+  }
+
+  &:first-of-type::before {
+    display: none;
+  }
+
+  & > div {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: center;
+    color: var(--color-content-light);
+    font-size: var(--paragraph-font-size-s);
+    font-weight: var(--font-weight-normal);
+  }
+
+  &:hover .pinWrapper {
+    display: block;
+  }
+}
+
+.text {
+  padding: 0px 16px 0px 5px;
+
+  &.capitalized span {
+    text-transform: capitalize;
+  }
+
+  & span {
+    width: 100%;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    font-weight: var(--font-weight-semi-bold);
+    font-size: var(--font-size-h6);
+  }
+}
+
+.expandToggle {
+  --content-font: 'basier-circle';
+
+  padding: 0px 5px !important;
```

Comment:
If you use nesting correctly, not only you don't need to prioritize using `!important` but also you prevent side effects.